threader3000.py

```diff
@@ -22,6 +22,7 @@
 def main():
     socket.setdefaulttimeout(0.30)
     print_lock = threading.Lock()
+    discovered_ports = [] # store discovered ports in here for later use
```

Comment:
This will be integrated with the next push, along with the accompanying functionality later in the code